packages/components/pages/f-mfa/src/services/EventBuilder.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable camelcase */
+// eslint-disable-next-line import/prefer-default-export
+export const buildEvent = (key, data = '') => {
+    switch (key) {
+        case 's1b':
```

Comment:
What are these keys? Did we make them up - could they be more obvious to somebody not involved?

Reply:
I was going to suggest adding to a constant file with the key describing what it is and the value as `s1b`